src/components/cc-warning-payment/cc-warning-payment.js

```diff
@@ -80,15 +97,24 @@ export class CcWarningPayment extends LitElement {
   /**
    * @param {PaymentMethodError} error
    */
-  _renderOrgaError ({ type }) {
+  _getOrgaError ({ type }) {
```

Comment:
Since it's not a subrender function, can we move it between the constructor and the main render?